chromium_presubmit_config.json5

```diff
@@ -153,6 +153,7 @@
     "CheckWebDevStyle": [
       "components/speedreader/resources/speedreader-desktop.css",
       "components/speedreader/resources/third_party/",
+      "components/webpack/gen-webpack-grd.js",
```

Comment:
nit: this is a regex, so: `.js` -> `\\.js`

Reply:
:facepalm: good catch